src/Parse.idr

```diff
@@ -54,10 +56,26 @@ sizedtok' tok = MkParameters tok (SizedList tok) (const $ pure ())
 Parser' : Type -> Nat -> Type
 Parser' = Parser TPState (sizedtok' Char)
 
+neComments : (Alternative mn, Monad mn, Subset Char (Tok p), Inspect (Toks p) (Tok p), Eq (Tok p)) =>
+           All (Parser mn p ())
```

Comment:
I'm not sure what the conventions are around line breaks and aligning type signatures, (I do think 120 char lines are ok btw) but I think some alignment wrt `:` would be nice. Let's ask about this at the dev meeting tomorrow.

Reply:
What I would usually do is to align the `A` on the second line with `(`